show/muxcable.py

```diff
@@ -412,3 +427,97 @@ def cableinfo(port):
 
     body = [[vendor, part_num]]
     click.echo(tabulate(body, headers=headers))
+
+
+@muxcable.group(cls=clicommon.AbbreviationGroup)
+def hwmode():
+    """start configuring cable in hardware mode on a port"""
+    pass
```

Comment:
should we be printing some error message related to missing params.

Reply:
I think click does print missing params by default. 

```admin@STR43-0101-0101-01LT0:~$ show mux hwmode 
Usage: show mux hwmode [OPTIONS] COMMAND [ARGS]...

  start configuring cable in hardware mode on a port

Options:
  -h, -?, --help  Show this message and exit.

Commands:
  mux-direction  Show muxcable mux_direction information```

Reply:
Yes, Click will print the help message for the current command if required params are missing. This is the beauty of using a third-part CLI library :)